Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "PProf"
 uuid = "e4faabce-9ead-11e9-39d9-4379958e3056"
 authors = ["Valentin Churavy <v.churavy@gmail.com>", "Nathan Daly <nhdaly@gmail.com>"]
-version = "1.0.0"
+version = "1.1.0"
```

Comment:
Is this a `1.1` or `2.0` change? We _are_ changing the default behavior, but not in any way that requires an API change on user's part (unless they don't want this new behavior, i guess).

Reply:
`1.1` is fine.